frontend/src/pages/article/components/article-view/styles.module.scss

```diff
@@ -39,10 +39,14 @@
 
 .iconButton {
   display: flex;
+  flex: 1 1 auto;
   justify-content: center;
   align-items: center;
-  width: 54px;
-  height: 54px;
+  min-width: 30px;
```

Comment:
I guess we can just remove `top: 100px;` from `buttonsWrapper`, so each button doesn't shrink in size. Currently the icons look quite large